packages/auth/src/platform_browser/recaptcha/recaptcha.ts

```diff
@@ -78,20 +82,58 @@ export class RecaptchaConfig {
   siteKey: string = '';
 
   /**
-   * The reCAPTCHA enablement status of the {@link EmailAuthProvider} for the current tenant.
+   * The list of providers and their enablement status for reCAPTCHA Enterprise.
    */
-  emailPasswordEnabled: boolean = false;
+  recaptchaEnforcementStateList: RecaptchaEnforcementState[] = [];
 
   constructor(response: GetRecaptchaConfigResponse) {
     if (response.recaptchaKey === undefined) {
       throw new Error('recaptchaKey undefined');
     }
     // Example response.recaptchaKey: "projects/proj123/keys/sitekey123"
     this.siteKey = response.recaptchaKey.split('/')[3];
-    this.emailPasswordEnabled = response.recaptchaEnforcementState.some(
-      enforcementState =>
-        enforcementState.provider === 'EMAIL_PASSWORD_PROVIDER' &&
-        enforcementState.enforcementState !== 'OFF'
+    this.recaptchaEnforcementStateList = response.recaptchaEnforcementState;
```

Comment:
To make it consistent with backend field names, how about renaming 

this.recaptchaEnforcementStateList -> this.recaptchaEnforcementState

interface RecaptchaEnforcementState -> interface RecaptchaEnforcementProviderState